sisyphus/global_settings.py

```diff
@@ -205,6 +205,9 @@ def file_caching(path):
 #: Show job targets on status screen, can significantly slow down startup time if many outputs are used
 SHOW_JOB_TARGETS = True
 
+# Controls whether to include 'creator' in serialized state of an abstract path
+INCLUDE_CREATOR_STATE = True
```

Comment:
I think we could even default to False and leave the option to be enabled for debugging..?

Reply:
I agree, I don't see how enabling it by default would cause any problems and if it does we can simply set it to True again.

Reply:
Makes sense, I changed it.